README.md

```diff
@@ -33,8 +36,6 @@ Give a ⭐️ if you like this project!
 - Hat tip to anyone whose code was used
 - Inspiration
 - Microverse
-- Net Ninja
-- Traversy Media
 
```

Comment:
- [ ] [OPTIONAL] I Kindly suggest that you add steps in your README file on how to run your scripts locally. This will help users know how to run your project.

## EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/rebaz36/LeaderBoard-API2.0/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/rebaz36/LeaderBoard-API2.0.git
```
```
- Once the files are on your machine, open the _LeaderBoard-API2.0_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:9001](http://localhost:9001) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```